source/computer/macos-setup.rst

```diff
@@ -237,30 +237,26 @@ Fortran 编译器，其提供了 ``gfortran`` 命令::
 Intel 软件开发工具包
 ^^^^^^^^^^^^^^^^^^^^
 
-`Intel oneAPI Toolkits <https://software.intel.com/content/www/us/en/develop/tools/oneapi.html>`__
-是 Intel 公司开发的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc`` 和 ``ifort`` 命令）。
-此外还有 MKL 数学库、MPI 并行库等。该工具包是免费的，不需要许可证。
+`Intel oneAPI <https://software.intel.com/content/www/us/en/develop/tools/oneapi.html>`__
+是 Intel 公司开发的软件开发工具包。其提供了 C/C++ 编译器（``icc`` 命令）和 Fortran 编译器（``ifort`` 命令），
+以及 MKL 数学库、MPI 并行库等众多软件开发工具。该工具包是免费的，不需要许可证。
 
-地震学新手可以先不安装此工具包，等日常科研中确实需要使用时再安装。
-
-.. dropdown:: :fa:`exclamation-circle,mr-1` 安装 Intel 软件开发工具包
-   :container: + shadow
-   :title: bg-info text-white font-weight-bold
-
-   在 macOS 下，官方手册提供了多种\
-   `安装方式 <https://software.intel.com/content/www/us/en/develop/documentation/installation-guide-for-intel-oneapi-toolkits-macos/>`__，
-   如在线安装、本地安装、使用 ``conda`` 安装等。这里，我们选择本地安装。
+.. note::
 
-   日常科研安装 Base Toolkit 和 HPC Toolkit 两个工具包即可。从官网下载\ `安装程序 <https://software.intel.com/content/www/us/en/develop/tools/oneapi/all-toolkits.html>`__。
-   选择 Local Installer，下载 :file:`.dmg` 文件并安装。默认安装目录是 :file:`/opt/intel/oneapi`。
+   地震学新手可以先不安装此工具包，等日常科研中确实需要使用时再安装。
 
-   配置环境变量::
+在 macOS 下，官方手册提供了\
+`多种安装方式 <https://software.intel.com/content/www/us/en/develop/documentation/installation-guide-for-intel-oneapi-toolkits-macos/>`__，
+如在线安装、本地安装、使用 ``conda`` 安装等。这里，我们推荐使用本地安装方式。
 
-       $ echo "source /opt/intel/oneapi/setvars.sh >/dev/null 2>&1" >> ~/.zshrc
+根据自己的需要，从官网下载
+`C/C++ 编辑器 <https://software.intel.com/content/www/us/en/develop/articles/oneapi-standalone-components.html#compilerclassic>`__
+或 `Fortran 编译器 <https://software.intel.com/content/www/us/en/develop/articles/oneapi-standalone-components.html#fortran>`__，
+的离线安装包（Offline），直接双击安装。默认安装目录是 :file:`/opt/intel/oneapi`。
 
-   更多设置可以参考\ `官方手册 <https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-intel-oneapi-base-hpc-macos/top.html>`__。
+安装完成后还需要配置环境变量::
 
-.. include:: intel-oneapi-warning.rst_
+    $ echo "source /opt/intel/oneapi/setvars.sh >/dev/null 2>&1" >> ~/.zshrc
 
```

Comment:
Perhaps we could add this link (https://software.intel.com/content/www/us/en/develop/articles/oneapi-standalone-components.html) in a dropdown similar to Linux so that macOS users can also find the local installers for other components.

Reply:
Done in 74be81f8975d9b8bce284590a037aeb3ee786193.